swagger/swagger.json

```diff
@@ -84,6 +84,58 @@
         }
       }
     },
+    "/api/curriculums/{curriculum_id}/cycles": {
+      "get": {
+        "summary": "Returns all Cycles",
+        "tags": [
+          "Cycles"
+        ],
+        "description": "Retrieves a list of all curriculum's cycles",
+        "parameters": [
+          {
+            "name": "curriculum_id",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "operationId": "getCycles",
+        "responses": {
+          "200": {
+            "description": "Success",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "integer"
+                      },
+                      "name": {
+                        "type": "string"
+                      },
+                      "learning_goals": {
```

Comment:
Please, run `rake rswag` to update the documentation file.

Reply:
Done!